configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionPreparer.java

```diff
@@ -112,6 +116,7 @@ public ConfigChangeActions prepare(SessionContext context, DeployLogger logger,
                                        Optional<ApplicationSet> currentActiveApplicationSet, Path tenantPath, 
                                        Instant now) {
         Preparation preparation = new Preparation(context, logger, params, currentActiveApplicationSet, tenantPath);
+        preparation.distributeApplicationPackage();
```

Comment:
I think so, but it depends on how it will be used and deployed, as you say. My plan was to end up building it the same way on all config servers, but there's much to do before we are in that state. Something to keep in mind